### ISSUE TITLE
QWebView in jlqml

### ISSUE THREAD

**Issue by behzaadh** [barche/jlqml#21]
Hello,

I wonder why you did not include QWebView in jlqml? any restriction that you tried to skip it?

**Comment by barche** [comment]
The need hasn't appeared until now. It should be easy to add, we just need a JLL for the relevant Qt library and add a wrapper for [`QtWebView::initialize()`](https://doc.qt.io/qt-6/qtwebview.html#initialize)

**Comment by behzaadh** [comment]
Yes, it is pretty easy by adding [QtWebView::initialize()](https://doc.qt.io/qt-6/qtwebview.html#initialize) to a wrapper. However, the Julia cannot load QTWebEngine library, even though it is compiled using MSVC 2019. Can you help me how I can create a JLL for the relevant Qt library? should be the same compiler x86_64-w64-mingw32 in windows or it can be MSVC 2019?

**Comment by barche** [comment]
It cannot work without a JLL package, and making one for QtWebView will need a few dependencies. I started the first PR on Yggdrasil here: https://github.com/JuliaPackaging/Yggdrasil/pull/8141

**Comment by barche** [comment]
This is going to be a little more difficult than anticipated, Qt WebView contains chromium, which is not so easy to cross-compile it seems. Progress is here: https://github.com/JuliaPackaging/Yggdrasil/pull/8229

Note that you can run the script from that PR locally to create a JLL package without  binaries and then use an `Overrides.toml` to point to your locally installed Qt, provided that you also build libcxxwrap_julia and jlqml (and depending on the platform Julia itself) from source, or if you feel up to it poke around with BinaryBuilder in `--debug` mode to try and complete the build.
